File: CS/RegistrationForm.BaseProject/ViewModel/RegistrationViewModel.cs
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Mvvm.POCO;

namespace RegistrationForm.ViewModel {
    [POCOViewModel]
    public class RegistrationViewModel {
        public static RegistrationViewModel Create() {
            return ViewModelSource.Create(() => new RegistrationViewModel());
        }
        protected RegistrationViewModel() { }
    }
}

File: VB/RegistrationForm.Lesson1/ViewModel/RegistrationViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations
Imports DevExpress.Mvvm.POCO

Namespace RegistrationForm.ViewModel

    <POCOViewModel>
    Public Class RegistrationViewModel

        Public Shared Function Create() As RegistrationViewModel
            Return ViewModelSource.Create(Function() New RegistrationViewModel())
        End Function

        Protected Sub New()
        End Sub
    End Class
End Namespace
